handlers.go

```diff
@@ -568,6 +568,14 @@ func (s *Server) handleAddCar(c echo.Context, u *User) error {
 		return err
 	}
 
+	defer func() {
+		go func() {
```

Comment:
Why are you using `defer` *and* goroutines here? If I understand correctly, `defer` will execute the code at the end of the function or if the function panics. Either way, I don't see why a goroutine here would be necessary.

Reply:
The call to `s.StagingMgr.CleanUp(bsid)` will take a while to return and we need not block the request because of it. That is why it is called in a goroutine